src/Service.GraphQLBuilder/Mutations/MutationBuilder.cs

```diff
@@ -130,7 +129,10 @@ string returnEntityName
                 switch (operation)
                 {
                     case EntityActionOperation.Create:
-                        mutationFields.Add(CreateMutationBuilder.Build(name, inputs, objectTypeDefinitionNode, root, databaseType, entities, dbEntityName, returnEntityName, rolesAllowedForMutation));
+                        // Get the point/batch fields for the create mutation.
+                        Tuple<FieldDefinitionNode, FieldDefinitionNode> createMutationNodes = CreateMutationBuilder.Build(name, inputs, objectTypeDefinitionNode, root, databaseType, entities, dbEntityName, returnEntityName, rolesAllowedForMutation);
+                        mutationFields.Add(createMutationNodes.Item1); // Add field corresponding to point insertion.
+                        mutationFields.Add(createMutationNodes.Item2); // Add field corresponding to batch insertion.
                         break;
```

Comment:
nit: lets not call it "batch" insertion. Lets use multiple/plural insertion to avoid confusion with batch inserts

Reply:
No need of that comment now.